src/models/StyleSheetManager.js

```diff
@@ -29,27 +26,34 @@ export default class StyleSheetManager extends Component<Props, void> {
     }),
   }
 
-  sheetInstance: StyleSheet
+  getContext: (sheet: ?StyleSheet, target: ?HTMLElement) => Object
+
+  constructor(props: Props) {
+    super(props)
 
-  getChildContext() {
-    return { [CONTEXT_KEY]: this.sheetInstance }
+    this.getContext = memoize(this.getContext.bind(this))
```

Comment:
small nit, I'd prefer it if instead of a constructor, we'd just wrap `getContext` inline, e.g.:


```js
  getContext = memoize((...) => {...});
```

Reply:
Yeah I agree that this looks a lot better, but after reading [articles like this](https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1) I had my concerns with perf among other things. But this might not be relevant anymore, not sure.